pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "sceptre-file-resolver"
+version = "1.0.6"
+description = "A Sceptre resolver to retrieve file content"
+authors = ["zaro0508 <zaro0508@gmail.com>"]
+readme = "README.md"
+homepage = "https://github.com/Sceptre/sceptre-file-resolver"
+license = "Apache-2.0"
+packages = [{ include = "resolver" }]
+keywords = ["sceptre", "sceptre-resolver"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Environment :: Console",
```

Comment:
you've setup the CI to execute tests for python 3.8-3.12 so maybe add `programming language` classifiers here?
```
  "Programming Language :: Python :: 3.8",
  "Programming Language :: Python :: 3.9",
  "Programming Language :: Python :: 3.10",
  "Programming Language :: Python :: 3.11"
  "Programming Language :: Python :: 3.12"
```

Reply:
These are included automatically by poetry based on the python requirement: https://python-poetry.org/docs/pyproject/#classifiers